service/docs/source/geopm_pio.3.rst

```diff
@@ -210,14 +212,20 @@ Inspection Functions
   Returns zero on success, error value on failure.
 
 ``geopm_pio_reset()``
-Reset the GEOPM platform interface causing resources to be freed.
-This will cause the internal PlatormIO instance to be
-released/deleted and reconstructed.  As a result, any signals and
-controls that had been pushed will be cleared, any batch servers
-that had been started will be stopped, and all registered IOGroups
-will be reset.  **NOTE: the reset only applies to the Service
-PlatformIO instance and does not affect the PlatformIO instance
-managed by the GEOPM HPC runtime.**
+  Reset the GEOPM platform interface causing resources to be freed.
+  This will cause the internal PlatormIO instance to be
+  released/deleted and reconstructed.  As a result, any signals and
+  controls that had been pushed will be cleared, any batch servers
+  that had been started will be stopped, and all registered IOGroups
+  will be reset.  **NOTE: the reset only applies to the Service
+  PlatformIO instance and does not affect the PlatformIO instance
+  managed by the GEOPM HPC runtime.**
```

Comment:
This comment is completely unrelated to this pull request, but this Note is misleading.  The reset applies to all use cases of PlatformIO with the one exception of the GEOPM HPC Runtime use case.  In the GEOPM HPC Runtime usecase, a different singleton access function is used, so this reset does not apply to the HPC Runtime PlatformIO instance.